fs/dafsclient/rmtclient.cpp

```diff
@@ -79,8 +79,26 @@ void cleanupDaFsSocket(ISocket *sock)
 ///////////////////////////
 
 
-static unsigned maxConnectTime = 0;
-static unsigned maxReceiveTime = 0;
+static const unsigned defaultDafsConnectTimeoutSeconds=100;
```

Comment:
NB: this was previously default to 0 (maxConnectTime), which caused connectSocket to use ISocket::connect without a timeout, however, ISocket::connect itself has a default timeout of 100 seconds.
Hence I have defaulted the dafilesrv connect timeout to 100 here.